src/fuzz_introspector/analyses/function_call_analyser.py

```diff
@@ -0,0 +1,256 @@
+
+    def analysis_func(
+        self,
+        toc_list: List[Tuple[str, str, int]],
+        tables: List[str],
+        proj_profile: project_profile.MergedProjectProfile,
+        profiles: List[fuzzer_profile.FuzzerProfile],
+        basefolder: str,
+        coverage_url: str,
+        conclusions: List[html_helpers.HTMLConclusion]
+    ) -> str:
+        logger.info(f" - Running analysis {Analysis.get_name()}")
+
+        # Getting data
+        callsite_list = []
+        function_list = []
+        for profile in profiles:
+            callsite_list.extend(cfg_load.extract_all_callsites(profile.function_call_depths))
+            for key in profile.all_class_functions.keys():
+                function_list.append(profile.all_class_functions[key])
+        (func_profile_list, called_func_dict, reachable_func_list) = (
+            self.third_party_func_profile(proj_profile, callsite_list, function_list)
+        )
+
+        html_string = ""
+        html_string += "<div class=\"report-box\">"
+
+        html_string += html_helpers.html_add_header_with_link(
+            "Function call coverage",
+            1,
+            toc_list
+        )
+
+        # Table with all function calls for each files
+        html_string += "<div class=\"collapsible\">"
+        html_string += (
+            "<p>"
+            "This section shows a list of 3rd party function "
+            "calls and their relative coverage information. "
+            "By static analysis of the target project code, "
+            "all of the 3rd party function call and their caller "
+            "information, including the source file and line "
+            "number that initiate the call are captured. "
+            "The caller source code file and line number are "
+            "shown in column 2 while column 1 is the function "
+            "name of the 3rd party function call. Each occurrent "
+            "of the 3rd party function call will occuply "
+            "a separate row. Column 3 of each row indicate if "
+            "the 3rd party call in the source file line is "
+            "unreachable. Column 4 lists all fuzzers that have "
+            "covered that particular system call in "
+            "that specific location (source file and line)" 
+            "during their dynamic fuzzing."
+            "</p>"
+        )
+
+        html_string += html_helpers.html_add_header_with_link(
+            "Function in each files in report",
+            2,
+            toc_list
+        )
+
+        # Third party function calls table
+        tables.append(f"myTable{len(tables)}")
+        html_string += html_helpers.html_create_table_head(
+            tables[-1],
+            [
+                ("Function name", ""),
+                ("Source file",
+                 "Source file and line number of this function call. "
+                 "Based on static analysis."),
+                ("Unused code? ",
```

Comment:
I think "reachable" is better here. Also, would it be possible to get runtime coverage information in as well?

Reply:
Runtime coverage in Column 4

Reply:
Changed to reachable code.